test/constructorio-catalog.js

```diff
@@ -0,0 +1,147 @@
+/* eslint-disable prefer-destructuring, no-unused-expressions */
+
+const expect = require('chai').expect;
+const fs = require('fs');
+const path = require('path');
+const Constructorio = require('../lib/constructorio');
+
+const testConfig = {
+  apiToken: 'YSOxV00F0Kk2R0KnPQN8',
```

Comment:
We should probably keep this not checked into the repository. In fact, because this has now been published to Github we should cycle the token.

Can we keep this in an .env file or pass through as an environment variable? Similar to how we do for `constructorio-client-javascript`. Checking in tokens to public repositories should be avoided.